test/data/resources-v0.json

```diff
@@ -0,0 +1,200 @@
+          "protocol": "lseed.",
+          "target": "1.2.3.4",
+          "priority": 0,
+          "weight": 5,
+          "port": 7868
+        }, {
+          "service": "a2.",
+          "protocol": "lseed.",
+          "target": "btcseeder",
+          "priority": 0,
+          "weight": 2,
+          "port": 1337
+        }],
+        "ttl": 64
+      }
+    },
+    "URI": {
+      "type": "URI",
+      "resource": {
+        "uri": ["ftp://ftp1.example.com/public"],
+        "ttl": 64
+      }
+    },
+    "EMAIL": {
+      "type": "RP",
+      "resource": {
+        "email": ["foo@gmail.com"],
+        "ttl": 64
+      }
+    },
+    "TEXT": {
+      "type": "TXT",
+      "resource": {
+        "text": ["this is a txt record"],
+        "ttl": 64
+      }
+    },
+    "LOCATION": {
+      "type": "LOC",
+      "resource": {
+        "location": [{
+          "version": 0,
+          "size": 0,
+          "horizPre": 1,
+          "vertPre": 2,
+          "latitude": 128,
+          "longitude": 256,
+          "altitude": 16
+        }],
+        "ttl": 64
+      }
+    },
+    "MAGNET": {
+      "type": "URI",
+      "resource": {
+        "magnet": [
+          "magnet:?xt=urn:btih:8c271f4d2e92a3449e2d1bde633cd49f64af888f"
+        ],
+        "ttl": 64
+      }
+    },
+    "DS": {
+      "type": "DS",
+      "resource": {
+        "ds": [{
+          "keyTag": 35215,
+          "algorithm": 13,
+          "digestType": 2,
+          "digest": "7C50EA94A63AEECB65B510D1EAC1846C973A89D4AB292287D5A4D715136B57A3"
+        }],
+        "ttl": 64
+      }
+    },
+    "TLS": {
+      "type": "TLSA",
+      "resource": {
+        "tls": [{
+          "protocol": "tcp.",
+          "port": 443,
+          "usage": 0,
+          "selector": 0,
+          "matchingType": 0,
+          "certificate": "0102348790"
+        }],
+        "ttl": 64
+      }
+    },
+    "SMIME": {
+      "type": "SMIMEA",
+      "resource": {
+        "smime": [{
+"hash": "92faa0601d44a3c2b98318d8f7e4cd3b17f25caa6753cc04365d753d",
+          "usage": 0,
+          "selector": 0,
+          "matchingType": 0,
+          "certificate": "85fc1da9c82de277abd69f65b962aa55c335697a8bb8418894b65f24d86c6de4"
+        }],
+        "ttl": 64
+      }
+    },
+    "SSH": {
+      "type": "SSHFP",
+      "resource": {
+        "ssh": [{
+          "algorithm": 0,
+          "keyType": 0,
+          "fingerprint": "9be3ba6df1bc232ad4642d33ac09b8dc991e9b76f4f74bc96dd2ff17aa94500d9af2bd8f73a9eee9"
+        }],
+        "ttl": 64
+      }
+    },
+    "PGP": {
+      "type": "OPENPGPKEY",
+      "resource": {
+        "pgp": [{
+          "hash": "0e9baa5e5ce8cffd7bf13c37423957ed5e0061c9c8807628979ef475",
+          "publicKey": "b796a94d827d3b3142dde13d945a0467b5c9afa53ae551bdf898b1f3101ef883"
+        }],
+        "ttl": 64
+      }
+    },
+    "ADDR": {
+      "type": "URI",
+      "resource": {
+        "addr": ["bitcoin:bc1qdsr2022f90pum2ptd8ftp4nvpgdhk656ue33qz"],
```

Comment:
Recommend using a regtest address for this, who knows... This address has history in the blockchain, belongs to somebody.

Reply:
Using both a mainnet and a testnet address that I generated locally